_includes/layouts/base.njk

```diff
@@ -26,18 +26,6 @@
     </header>
 
     <main{% if templateClass %} class="{{ templateClass }}"{% endif %}>
-
```

Comment:
Unrelated change:
This yellow notice was distracting and unpleasant to look at. I removed it.

Reply:
yes, it was aweful!!!!!!!!!